### PR TITLE
Attribute edits to check for standard units 

### PR DESCRIPTION
**Pull request by ErinCain**

### CODE REVIEW

File: R/attribute-element.R [hunk 1]
```diff
@@ -357,16 +357,30 @@ add_interval_ratio <- function(type = c("interval", "ratio"), units, unit_precis
...
       stop(interval_error_message, call. = FALSE)
     } 
   } 
   type <- match.arg(type)
   
-  interval_ratio <- list(unit = list(standardUnit = units),
-                         precision = unit_precision,
-                         numericDomain =
-                           list(numberType = number_type))
+  if (units %in% standard_units) {
```
Comment (thesadie) [inline R/attribute-element.R]: should `standard_units` have the package namespace `EMLaide::standard_units`? i like that style better, lets me know where something is coming from

**Review by thesadie — COMMENTED** [review]
looks good. did you build the docs too?

File: R/attribute-element.R (hunk 1 above)
Comment (ErinCain) [reply R/attribute-element.R]: Agreed! Changed to EMLaide::standard_units

**Comment by ErinCain** [comment]
Added namespace to `standard_units` and rebuilt all docs. Then merged

**Pull request merged**